tests/test_manifest_audit.py

```
@@ -0,0 +1,333 @@
+    from kb_setup.result import Rc
+
+    assert manifest_audit.main(tmp_path, ["--bogus"]) == int(Rc.BAD_REQUEST)
+
+
+def test_main_writes_the_sidecar_gates_reads_back(tmp_path: Path, monkeypatch) -> None:
+    """The channel C3 asked for: `main` writes what `gates.read_sidecar_outcome` reads."""
+    import subprocess
+
+    monkeypatch.chdir(tmp_path)
+    subprocess.run(["git", "init", "-q"], cwd=tmp_path, check=True)
+    subprocess.run(["git", "commit", "--allow-empty", "-q", "-m", "root"], cwd=tmp_path, check=True)
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)